src/api/handlers/translation.py

```diff
@@ -0,0 +1,31 @@
+from _main_.utils.massenergize_response import MassenergizeResponse
+from _main_.utils.route_handler import RouteHandler
+from api.decorators import admins_only, super_admins_only
+from api.services.translation import TranslationsService
+from _main_.utils.context import Context
+
+
+class TranslationsHandler(RouteHandler):
+	def __init__(self):
+		super().__init__()
+		
+		self.service = TranslationsService()
+		self.registerRoutes()
+	
+	def registerRoutes(self):
+		self.add("/translations.languages.get", self.get_all_languages)
```

Comment:
```suggestion
		self.add("/translations.supported_languages.list", self.list_all_supported_languages)
```

Reply:
This endpoint is meant to get all languages(from a json) that an admin can add to the database as supported languages. @archx3 will create an endpoint for fetching all currently supported languages from the database.

Reply:
if you are getting a listed of languages, the api should end with list.